docs/data_preparation.md

```diff
@@ -16,10 +16,15 @@ MMPose supported datasets:
 - [FreiHand](https://lmb.informatik.uni-freiburg.de/projects/freihand/)
 - [CMU Panoptic HandDB](http://domedb.perception.cs.cmu.edu/handdb.html)
 - [InterHand2.6M](https://mks0601.github.io/InterHand2.6M/)
+- [Human3.6M](http://vision.imar.ro/human3.6m/description.php)
+- [MPI-INF-3DHP](http://gvv.mpi-inf.mpg.de/3dhp-dataset/)
```

Comment:
Which dataset type should these new datasets use?

Reply:
You see, we have many specific dataset classes

![image](https://user-images.githubusercontent.com/9464825/99925723-28e09900-2d7a-11eb-995f-24b269b7a7ce.png)

Which class should be used for those new datasets?

Reply:
These datasets are used in `MeshMixDataset` see   https://github.com/open-mmlab/mmpose/blob/f456debefab5d8b29bbf62785f76077a0b9b4a12/configs/mesh/hmr/hmr_resnet_50.py#L116

Reply:
When training HMR, many datasets are used, including:
h36m, mpi_inf_3dhp, lsp, lspet, mpii, coco2014

Reply:
okay